src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas1/Cas1SpaceBookingTest.kt

```diff
@@ -459,7 +459,7 @@ class Cas1SpaceBookingTest {
     }
 
     @Test
-    fun `Search with no filters excludes departed bookings`() {
+    fun `Search with no filters excludes no cancelled bookings`() {
```

Comment:
i think we have a double negative here, should it be `Search with no filters excludes cancelled bookings`